src/Fade.js

```diff
@@ -50,12 +51,12 @@ function Fade(props) {
     children,
     innerRef = ref,
     ...otherProps
-  } = props;
+  } = addDefaultProps(fadeDefaultProps, props);
 
-  const transitionProps = pick(
-    { defaultProps, ...otherProps },
+  const transitionProps = pick(otherProps,
```

Comment:
@amillward I changed this line to remove the `defaultProps` prop. 
I was assuming the previous version was a mistake because `defaultProps` would be passed-in as prop. I could not find a hint in the code or the documentation for `Transition` that this prop is supported. There is no change in the test results either. Can you confirm that this change is ok?

Reply:
Actually, it would be immediately removed by the `pick` function.